docs/source/markdown/podman-build.1.md

```diff
@@ -540,6 +540,15 @@ layers are not squashed.
 Squash all of the new image's layers (including those inherited from a base
 image) into a single new layer.
 
+#### **--ssh**=*default|id[=socket>|[,]*
```

Comment:
could we move this chunk to a separate commit as well as a separate commit for the buildah version bump?

Reply:
@giuseppe `Build commit test` fails since original commit is dependent upon updated builah. We could create a seperate PR for version bump and merge that first.